### PR TITLE
use link-to instead of a tag

### PR DESCRIPTION
**Pull request by hakubo**
todo: 
- [x] url should be in form of 'Namespace:Title' instead of '/wiki/Namespace:Title'

### CODE REVIEW

File: front/main/app/models/mediawiki/category.js [hunk 1]
```diff
@@ -27,8 +29,42 @@ function getUrlBatchContent(categoryName, index, batch) {
...
 }
+/**
+ * add title to collectionItem based on url eg. /wiki/Namespace:Title -> Namespace:Title
+ *
+ * @param  {Array.<{url: string, name: string}>} collectionItems - array of items
+ * @returns {Array.<{url: string, name: string, title: string}>}
+ */
+function addTitlesToCollection(collectionItems) {
+	return collectionItems.map((item) => {
+		item.title = item.url.replace('/wiki/', '');
```
Comment (rogatty) [inline front/main/app/models/mediawiki/category.js]: Does it make sense to be safer and use `new RegExp('^/wiki/')`?

File: front/main/app/models/mediawiki/category.js [hunk 2]
```diff
@@ -28,8 +30,42 @@ function getUrlBatchContent(categoryName, index, batch) {
...
+
+		return item;
+	});
+}
+
+/**
+ * Adds titles to collection
```
Comment (rogatty) [inline front/main/app/models/mediawiki/category.js]: Can you also explain when `collections` param is an object and when it's an array? This doc isn't really helpful, especially that there is another function called `addTitlesToCollection`.

File: front/main/app/models/mediawiki/category.js (hunk 1 above)
Comment (hakubo) [inline front/main/app/models/mediawiki/category.js]: string replace replaces only first occurrence

Comment (rogatty) [inline front/main/app/models/mediawiki/category.js]: I guess we can safely assume that there is always `/wiki/` at the beginning and not worry about later occurrences, right?

**Comment by rogatty** [comment]
LGTM

File: front/main/app/models/mediawiki/category.js (hunk 1 above)
Comment (hakubo) [inline front/main/app/models/mediawiki/category.js]: we always send local path to client and all of our wikis have /wiki/ as article path

PS. this probably should be moved to api - now when I think about that. 
but would go with that for now

File: front/main/app/models/mediawiki/category.js [hunk 3]
```diff
@@ -51,32 +87,21 @@ const CategoryModel = Ember.Object.extend({
...
 		const url = getUrlBatchContent(this.get('name'), index, batchToLoad);
 
-		return Ember.$.ajax({
+		return $.ajax({
 			url,
 			dataType: 'json',
 			method: 'get',
 		}).then((pageData) => {
-			this.set(
-				`collections.${index}.items`,
```
Comment (bognix) [inline front/main/app/models/mediawiki/category.js]: ticket for that already created you can annotate this method with @TODO - https://wikia-inc.atlassian.net/browse/XW-1165

File: front/main/app/models/mediawiki/category.js (hunk 1 above)
Comment (bognix) [inline front/main/app/models/mediawiki/category.js]: ticket for that already created you can annotate this method with @TODO - https://wikia-inc.atlassian.net/browse/XW-1165

**Pull request merged**